resources/processor-types/log_dedup.yaml

```
@@ -0,0 +1,51 @@
+apiVersion: bindplane.observiq.com/v1
+kind: ProcessorType
+metadata:
+  name: log_dedup
+  displayName: DeDuplicate Logs 
```

Thoughts on changing capitalization here? 
```suggestion
  displayName: Deduplicate Logs 
```